src/main/kotlin/no/nav/familie/tilbake/forvaltning/ForvaltningService.kt

```diff
@@ -0,0 +1,54 @@
+package no.nav.familie.tilbake.forvaltning
+
+import no.nav.familie.tilbake.behandling.BehandlingRepository
+import no.nav.familie.tilbake.common.exceptionhandler.Feil
+import no.nav.familie.tilbake.common.repository.findByIdOrThrow
+import no.nav.familie.tilbake.kravgrunnlag.HentKravgrunnlagService
+import no.nav.familie.tilbake.kravgrunnlag.KravgrunnlagRepository
+import no.nav.familie.tilbake.kravgrunnlag.domain.KodeAksjon
+import no.nav.familie.tilbake.kravgrunnlag.ØkonomiXmlMottattService
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.http.HttpStatus
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import java.math.BigInteger
+import java.util.UUID
+
+@Service
+class ForvaltningService(private val behandlingRepository: BehandlingRepository,
```

Comment:
Metodene her opererer bare på kravgrunnalg, så de burde ha ligget i KravgrunnlagService.

Reply:
Denne tjenesten kommer til å utvide for å implementere andre forvaltnings deler også. De tjenestene kommer vi ikke til å bruke fra frontend og kun brukes av forvaltningsteam via swagger. Derfor er det fornuftig til å skille den fra andre delene av tilbakekreving, tror jeg.